care/emr/reports/context_builder/report_builder.py

```
@@ -0,0 +1,61 @@
+from care.emr.reports.context_builder.builders import (  # noqa: F401
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)